Sources/Rendering/Core/VolumeMapper/index.js

```diff
@@ -2,9 +2,45 @@ import macro from 'vtk.js/Sources/macros';
 import * as vtkMath from 'vtk.js/Sources/Common/Core/Math';
 import Constants from 'vtk.js/Sources/Rendering/Core/VolumeMapper/Constants';
 import vtkAbstractMapper from 'vtk.js/Sources/Rendering/Core/AbstractMapper';
+import vtkPiecewiseFunction from 'vtk.js/Sources/Common/DataModel/PiecewiseFunction';
+import vtkColorTransferFunction from 'vtk.js/Sources/Rendering/Core/ColorTransferFunction';
 
 const { BlendMode, FilterMode } = Constants;
 
+function createRadonTransferFunctions(values) {
+  const ofun = vtkPiecewiseFunction.newInstance();
+  ofun.addPoint(-3024, 0.1);
+  ofun.addPoint(-637.62, 0.1);
+  ofun.addPoint(700, 0.5);
+  ofun.addPoint(3071, 0.9);
```

Comment:
why is it hard-coded and does not depend on `values` ?

Reply:
These values already been set before. So we let it as it was.